tests/dummy/app/pods/radio/template.hbs

```diff
@@ -0,0 +1,58 @@
+<div class="dummy-notification-container">
+{{#each notifications as |notification|}}
+  {{notification-message notification=notification}}
+{{/each}}
+</div>
+
+<div class='dummy-body checkbox'>
+
+  <div class="section-title">Frost Radio Button</div>
+  <hr>
+
+  <div class='section'>
+
+    <div class='example'>
+      <div class='demo'>
+        {{#each sampleList as |i|}}
+          {{#frost-radio-button
+            value=i
+            groupValue=groupName
+            changed='change'
```

Comment:
Can you add an example here of currying the groupId into the event params?

e.g. `onChange=(action 'change' groupId)`